python/hail/keytable.py

```diff
@@ -1503,12 +1503,26 @@ def maximal_independent_set(self, i, j):
         Prune individuals from a dataset until no close relationships remain
         with respect to a PC-Relate measure of kinship.
 
-        >>> related_pairs = hc.import_vcf("data/sample.vcf.bgz").pc_relate(2, 0.001).filter("kin > 0.125")
+        >>> vds = hc.import_vcf("data/sample.vcf.bgz")
+        >>> related_pairs = vds.pc_relate(2, 0.001).filter("kin > 0.125")
         >>> related_samples = related_pairs.query('i.flatMap(i => [i,j]).collectAsSet()')
         >>> related_samples_to_keep = related_pairs.maximal_independent_set("i", "j")
         >>> related_samples_to_remove = related_samples - set(related_samples_to_keep)
         >>> vds.filter_samples_list(list(related_samples_to_remove))
 
+        Prune individuals from a dataset, prefering to keep cases over controls.
+
+        >>> vds = hc.read("data/example.vds")
+        >>> related_pairs = vds.pc_relate(2, 0.001).filter("kin > 0.125")
+        >>> related_samples = related_pairs.query('i.flatMap(i => [i,j]).collectAsSet()')
+        >>> related_samples_to_keep = (related_pairs
+        ...   .key_by("i").join(vds.samples_table()).annotate('iAndCase = { id: i, isCase: sa.isCase }').select('iAndCase')
+        ...   .key_by("j").join(vds.samples_table()).annotate('jAndCase = { id: j, isCase: sa.isCase }').select('jAndCase')
```

Comment:
this is breaking the docs since you've got the `.select('iAndCase')` before the `.key_by("j")`?